.github/workflows/publish-next.yml

```diff
@@ -49,3 +49,9 @@ jobs:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
       - name: Publish to next tag
         run: yarn publish:next
+      - name: Merge main into docs-release branch
+        if: steps.changesets.outputs.published == 'true'
+        run: |
+          git checkout docs-release
```

Comment:
What kind of protections should we put around this branch since deploying to it can take down our site? We probably shouldn't be able to push to it (wish exception of github action)